### ISSUE TITLE
Curious why F.normalize() is commented in the training script?

### ISSUE THREAD

**Issue by ChloeL19** [NVIDIA/ngpt#7]
Curious about the question in the title. Wonder if you discovered an instability or inefficiency with F.normalize() instead of the other norm function?
